packages/scaffold-config/src/frameworks.ts

```diff
@@ -61,6 +61,22 @@ function getBundlerDependency (bundler: WizardBundler['type'], projectPath: stri
   }
 }
 
+export const WIZARD_MOUNT_MODULES = ['cypress/react', 'cypress/react18', 'cypress/vue', 'cypress/vue2', 'cypress/angular'] as const
+
+export type WizardMountModule = typeof WIZARD_MOUNT_MODULES[number]
+
+const mountModule = (mountModule: WizardMountModule) => async (projectRoot: string) => Promise.resolve(mountModule)
```

Comment:
```suggestion
const mountModule = (mountModule: WizardMountModule) =>  (projectRoot: string) => Promise.resolve(mountModule)
```

I think you can remove `async` here, implied by the `Promise.resolve`